src/relay/backend/aot_executor_codegen.cc

```diff
@@ -1328,17 +1333,11 @@ class AOTExecutorCodegenModule : public runtime::ModuleNode {
   runtime::NDArray get_param_by_name(String key) {
     auto it = this->output_.params.find(key);
     CHECK(it != this->output_.params.end()) << "no such parameter " << key;
-    return (*it).second.second;
+    return (*it).second;
```

Comment:
huh?

Reply:
The params (constants) map was capturing both the ndarray and the internal id, but the id was never used. So I removed the id. This means the constants map is always string -> ndarray so the "const_name_to_ndarray" attribute can be used consistently irrespective of executor. It also means there's no need for the extra projection.